cirq-ionq/cirq_ionq/ionq_gateset.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Target gateset used for compiling circuits to IonQ device."""
+from typing import Any
+from typing import Dict
+from typing import List
+
+import cirq
+from cirq.transformers.target_gatesets.compilation_target_gateset import (
+    _create_transformer_with_kwargs,
```

Comment:
Should we be making this public?  This creates a dependency on into cirq that is "private" and so subject to change.

Reply:
agreed, filed https://github.com/quantumlib/Cirq/issues/5491 to track this change.